src/utils/debug.ts

```diff
@@ -0,0 +1,14 @@
+/** Find the name of the calling component. */
+export const getHookParent = () => {
```

Comment:
Nice! 👏 naming of the file might not be optimal but that's alright. Can we make sure that NODE_ENV is used here so it becomes a noop in prod?

Reply:
So I was using the NODE_ENV checks when using the react stack trace but I don't see any advantage to hiding these details in prod as they'll just be ignored.

Reply:
It's mainly so that this compiles away in a user's production bundle since it'd otherwise be dead code 👍

Reply:
Fair enough - I'll pull this into a hook also then :+1: